frontend/app/component/election/ElectionStatusWithIcon.test.tsx

```diff
@@ -0,0 +1,102 @@
+import { render } from "@testing-library/react";
+import { describe, expect, test } from "vitest";
+
+import { ElectionStatusWithIcon } from "./ElectionStatusWithIcon";
+
+describe("ElectionStatusWithIcon", () => {
+  test("renders the correct header coordinator status for DataEntryInProgress", () => {
+    const { getByText, getByRole } = render(ElectionStatusWithIcon("DataEntryInProgress", true, true));
+
+    expect(getByText("Steminvoer bezig")).toBeVisible();
+    expect(getByText("(eerste zitting)")).toBeVisible();
+    expect(getByRole("img")).toBeVisible();
```

Comment:
We'll have to do something with either `title` or `alt text`, so we can use `getByTitle` or `getByAltText`.

Or we can use the `alt` attribute in combination with `toHaveAccessibleName()`. For some other icons we use `toHaveAccessibleName()` and `aria-label`, but there we need the `aria-label` to convey information that's otherwise not available. Here we have text plus icon, so the icon is 'just' an extra.

@jorisleker What can we do with this icon, while avoiding that a screen reader first reads the text and then also 'reads' the icon?
![image](https://github.com/user-attachments/assets/db2d1941-e593-41e8-92cc-9a81852a7e23)

Reply:
I can't find anything about React Testing Library being able to check if the icon is in front or behind the text. I'm not even sure if RTL renders the DOM in a way that it could know.

Playwright doesn't seem to be able either, see this [bug](https://github.com/microsoft/playwright/issues/9923). The closest you can get is clicking with `{ trial: true }` on all four corners of the image (using x/y coordinates), because the click will fail if something else is on top of the image at those points.

So I don't think it makes sense to assert on this.

Reply:
@jschuurk-kr @oliver3 What if we do this?

```html
<img src="icon.svg" alt="icon description" aria-hidden="true" />
```

`Aria-hidden` should instruct screenreaders that the image (and it's alt text) should be ignored. And I don't think testing wether the icon is before or behind the text makes much sense. Bit too much detail?

Reply:
Talking to @jschuurk-kr we came with the following suggestion
```typescript
  test("renders the correct header coordinator status for DataEntryInProgress", () => {
    const { container } = render(ElectionStatusWithIcon("DataEntryInProgress", true, true));

    expect(container.children.length).toBe(2);
    expect(container.children[0]).toHaveTextContent("Steminvoer bezig (eerste zitting)");
    const icon = within(container.children[1] as HTMLElement).getByRole("img");
    expect(icon).toHaveAttribute("data-icon", "IconCheckHeart");
}
```